usage-metrics/Makefile

```
@@ -0,0 +1 @@
+include ../commons-test.mk
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)